### ISSUE TITLE
Can't find packages (breaks the installation process)

### ISSUE THREAD

**Issue by HelifeWasTaken** [Epitech/dump#14]
Thoses packages from the package list cannot be found on some computer with dnf :   
```
             x264.x86_64
             lightspark.x86_64
             lightspark-mozilla-plugin.x86_64
```
The process is simply launching the script on two pc that has the exact same Fedora 32 raw installation with XFCE but not the same hardware.

What seems to occure is that depending on the hardware some packages can't be found on some pc (Technical reason unknown)

The problem that gives thoses broken packages is : 
```
g++ cmake make gcc git SFML SFML-devel (and 100 more packages are skipped)
```
:( Is there an alternative to thoses packages or should we always remove them by hand if dnf does not find them

EDIT : I found the REAL issue since rpm fusion repositories are already enabled

**Comment by HelifeWasTaken** [comment]
For now to fix the missing packages I just installed everything with a little script like shown beneath before running the script again
```sh
    echo "Updating all packages to the latest available version (i.e. doing dnf upgrade)..."
    dnf upgrade -y

    local packages=$(curl https://raw.githubusercontent.com/Epitech/dump/master/install_packages_dump.sh | grep -oPz 'packages_list=\(\K[^\)]+' | sed 's/x264.x86_64//g' | sed 's/lightspark.x86_64//g' | sed 's/lightspark-mozilla-plugin.x86_64//g')

    if [ -z "$packages" ]; then
        echo "Could not get the package list" 2>&1
        echo "Please report this to the repo manager by creating an issue" 2>&1
        exit 1
    fi

    echo "Installing packages from Epitech dump package list..."
    dnf -y install $packages
```

**Comment by GabrielRavier** [comment]
These packages are from RPM Fusion, a software provider that provides software that the Fedora Project or Red Hat does not want to ship themselves (i.e. it is non-free or patent-encumbered, which I believe is the case for x264 and lightspark, being patent encumbered in relation to the H.264 and SWF formats).

You can make it possible to install these packages by [enabling RPM Fusion as a package provider](https://rpmfusion.org/Configuration/), which you could do here with a command line such as `dnf install https://mirrors.rpmfusion.org/free/fedora/rpmfusion-free-release-$(rpm -E %fedora).noarch.rpm`, which installs RPM Fusion's free repository as a package provider (x264 and lightspark are from the free repo, so no need to install the non-free repos onto people who most likely would not want them).

PS: IDK if you care but the `.x86_64` at the end of the package name have no purpose apart from specifying that you specifically want to install the `x86_64` version of the package. By default, the version matching your system would be installed, making this basically useless unless your goal is to specifically install the AMD64 version of a package onto a non-AMD64 architecture (I personally cannot see why this would be useful here).

**Comment by HelifeWasTaken** [comment]
Interesting I did not thought about searching for rpm fusion packages I assumed that the dump could be runned in a raw version of Fedora

That might be certainly the problem ...

**Comment by HelifeWasTaken** [comment]
Just tried to enable rpm Fusion non free update repositories and it works !

**Comment by HelifeWasTaken** [comment]
Just noticed !

```
dnf -y install dnf-plugins-core && dnf -y copr enable petersen/stack2 && dnf -y install https://download1.rpmfusion.org/free/fedora/rpmfusion-free-release-$(rpm -E %fedora).noarch.rpm https://download1.rpmfusion.org/nonfree/fedora/rpmfusion-nonfree-release-$(rpm -E %fedora).noarch.rpm
```
had already rpm fusion

But since we use `&&` operator and `dnf -y copr enable petersen/stack2` fail with this error : 
```
This repository does not contain any executables yet so you cannot activate it
```
It does not active rpm fusion and everything breaks :(

**Comment by GabrielRavier** [comment]
Why do we need the nonfree repos ? x264 and lightspark are from the free repos.

**Comment by HelifeWasTaken** [comment]
#15 Pull request that fixes the dump

PS: Also wondering about non free repositories 🤔

**Comment by Xephi** [comment]
Fixed by #15
